src/Dafda.Tests/Producing/TestOutgoingMessageFactory.cs

```diff
@@ -27,7 +27,7 @@ public async Task Can_create_outgoing_message_from_registry_with_expected_raw_me
 
             await sut.Produce(new DummyMessage(dummyAggregateId));
 
-            Assert.Equal($@"{{""messageId"":""{dummyMessageId}"",""type"":""{DummyType}"",""data"":{{""aggregateId"":""{dummyAggregateId}""}}}}", spy.Value);
+            Assert.Equal($@"{{""messageId"":""{dummyMessageId}"",""type"":""{DummyType}"",""causationId"":""foo_id"",""correlationId"":""foo_id"",""data"":{{""aggregateId"":""{dummyAggregateId}""}}}}", spy.Value);

```

Comment:
📐 This value is getting a bit long and hard to read now, any chance we could break it out to a named value and spread it over a few lines so it's clearer to see what data is in it?